public/xmlns/content.xsd

```diff
@@ -93,6 +93,15 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:simpleType name="textStyle">
+        <xs:restriction base="xs:token">
+            <xs:enumeration value="bold" />
```

Comment:
are there any besides **bold** that it makes sense to add now while we're at it?

Reply:
`italic`? or should it be `italics`

Reply:
`italic`

Reply:
_italic_ and underline ?